documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
+.Procedure
+Execute the following steps for each `Kafka.spec.cruiseControl` configuration in your Kafka cluster.
+
+. Update the `Kafka` resource in an editor.
```

Comment:
Do we want to use _custom resource_?

Reply:
Yes, I think so. Changed to custom resource in each Kafka-related procedure.